data/report_templates/project_summary.md

```diff
@@ -87,53 +90,74 @@ Data Processing
 Data Analysis
 :   {{ project.accredit.data_analysis }}
 
-# Sample Info
-
-NGI ID | User ID | Mreads | >=Q30(%) | Status
--------|---------|--------|----------|--------
-{% for sample in samples.values()  -%}
-{{ sample.ngi_id }} | {{ sample.customer_name }} | `{{ sample.total_reads }}` | {{ sample.qscore }} | {{ sample.seq_status }}
-{%- endfor %}
+# Sample Information
+{% if not samples %}
+No sample information to be displayed.
+{% elif samples|length > project.display_limit %}
+There are too many samples to display, please click [here]({{ project.ngi_name }}_sample_info.txt) to view the table from tab-separated text file. Below you can find an explanation of the header column used in the table.
+
+{{ tables.sample_info }}
+{% else %}
+NGI ID | User ID | Mreads | >=Q30(%) {% if project.ordered_reads %}| Status {% endif %}
+-------|---------|--------|----------{% if project.ordered_reads %}|-------- {% endif %}
+{% for sample in samples.values() -%}
+{{ sample.ngi_id }} | {{ sample.customer_name }} | `{{ sample.total_reads }}` | {{ sample.qscore }} {% if project.ordered_reads %} | {{ sample.seq_status }} {% endif %}
+{% endfor %}
+
```

Comment:
Can we still link to the `_sample_info.txt` file even when we have < 10 samples? People might find it useful.